backend/app/assets/javascripts/spree/backend/turbolinks-configuration.js.erb

```diff
@@ -0,0 +1,20 @@
+// comment line necessary for correct interpolation of use_turbolinks
+Turbolinks.supported = <%= Spree::Backend::Config.use_turbolinks %>;
```

Comment:
Instead would it be possible to do by having a user edit their `vendor/assets/javascripts/spree/backend/all.js`?